Header header
bool battery_low
bool battery_lower
int8 battery_percentage
bool battery_state
int8 camera_state
bool down_visual_state
int16 drone_battery_left
int16 drone_fly_time_left
bool drone_hover
bool em_open
bool em_sky
bool em_ground
int16 east_speed
int16 electrical_machinery_state
bool factory_mode
int8 fly_mode
int16 fly_speed
int16 fly_time
bool front_in
bool front_lsc
bool front_out
bool gravity_state
int16 ground_speed
int16 height
int8 imu_calibration_state
bool imu_state
int8 light_strength
int16 north_speed
bool outage_recording
bool power_state
bool pressure_state
int16 smart_video_exit_mode
bool temperature_height
int8 throw_fly_timer
int8 wifi_disturb
int8 wifi_strength
bool wind_state
